packages/svelte-flicking/src/Flicking.js

```diff
@@ -0,0 +1,9 @@
+import { withFlickingMethods } from "@egjs/flicking";
+
+import SvelteFlicking from "./flicking.svelte";
+
+export default (() => {
+  withFlickingMethods(SvelteFlicking, "vanillaFlicking");
```

Comment:
Shouldn't this method access the prototype?

Reply:
You're right. I believe I've put prototype in that, wonder why it's gone.